test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,23 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(500);
```

Comment:
`100`, although really you could probably get away with something smaller. If this is taking more than 50 or so `ms` you've got some big issues.